pandas/tests/arithmetic/conftest.py

```diff
@@ -28,14 +28,32 @@ def zero(request):
     return request.param
 
 
+# ------------------------------------------------------------------
+# Vector Fixtures
+
 @pytest.fixture(params=[pd.Float64Index(np.arange(5, dtype='float64')),
                         pd.Int64Index(np.arange(5, dtype='int64')),
-                        pd.UInt64Index(np.arange(5, dtype='uint64'))],
+                        pd.UInt64Index(np.arange(5, dtype='uint64')),
+                        pd.RangeIndex(5)],
                 ids=lambda x: type(x).__name__)
-def idx(request):
+def numeric_idx(request):
+    """
+    Several types of numeric-dtypes Index objects
+    """
     return request.param
 
 
+@pytest.fixture
+def tdser():
```

Comment:
should conform / rename these at some point (e.g. have been using: timedelta_series) for something like this